app/models/state_file_nc_intake.rb

```diff
@@ -150,4 +150,8 @@ def disqualifying_eligibility_rules
   def show_tax_period_in_return_header?
     false
   end
+
+  def requires_additional_debit_information?
```

Comment:
[dust] For the most part we've been referring to electronic taxpayer payments as "withdrawals", so I have a very mild preference to stay consistent instead of use "debit"